bubblesub/cmd/set_volume.py

```diff
@@ -18,13 +18,21 @@
 
 from bubblesub.api import Api
 from bubblesub.api.cmd import BaseCommand
+from bubblesub.api.gui import ViewState
 from bubblesub.util import eval_expr
 
 
 class SetVolumeCommand(BaseCommand):
     names = ["set-volume"]
     help_text = "Adjusts the video volume."
 
+    @property
+    def is_enabled(self) -> bool:
+        return (
+            self.api.gui.current_view == ViewState.Audio
+            or self.api.gui.current_view == ViewState.Full
+        )
```

Comment:
This doesn't make sense. Why can't you raise the volume of the played lines just because the spectrogram is hidden?